interceptor/main.go

```diff
@@ -90,6 +90,7 @@ func main() {
 		lggr,
 		cl,
 		servingCfg.ConfigMapCacheRsyncPeriod,
+		servingCfg.CurrentNamespace,
```

Comment:
Can you elaborate on this please?

Reply:
Sure! When I changed the permissions so the add-on components only had access to ConfigMaps on the `keda` namespace, it started to fail with the following error:

```log
W0224 02:33:59.494889       1 reflector.go:424] pkg/mod/k8s.io/client-go@v0.26.1/tools/cache/reflector.go:169: failed to list *v1.ConfigMap: configmaps is forbidden: User "system:serviceaccount:keda:keda-http-add-on-interceptor" cannot list resource "configmaps" in API group "" at the cluster scope
```

After investigating, I found that the ConfigMap Informer was watching all ConfigMaps of the cluster, not just the ones on the `keda` namespace. Then I changed it to only look at ConfigMaps from the `keda` namespace (https://github.com/kedacore/http-add-on/pull/613/files#diff-a2bfc341efdd78af356e3724892bfd7ef8c5cfb3cc9cf14b209df74db39249b8L129-L132). But a user may want to install KEDA on another namespace and this is parameterised, so I need to tell the informer which namespace the add-on is installed for it to watch at ConfigMaps there. That's why I had to add this extra parameter.